hooks/useLobbyActions.ts

```diff
@@ -48,10 +48,17 @@ export default function useLobbyActions() {
     update(roomRef, { status });
   }
 
+  async function onCopyLink(){
+    const { roomId } = router.query;
+    const urlString = "https://saydontsay.netlify.app/room/" + roomId;
```

Comment:
urlString should be dynamic (fetch it on call) instead of hardcoded. By doing that, we can change our website's domain without needing to change the code. Also you don't need to make the url string by hand, you are in lobby page so you can just copy the current url.